crates/mpz-garble/src/protocol/semihonest/evaluator.rs

```diff
@@ -217,6 +221,18 @@ where
         Ok(())
     }
 
+    fn wants_preprocess(&self) -> bool {
```

Comment:
Will there be a situation where the first call in the call_stack is not ready for preprocessing?
Otherwise, this fn can just be
```!self.call_stack.is_empty()```

Reply:
opened https://github.com/privacy-scaling-explorations/mpz/issues/277